src-ghc/Pact/Types/Server.hs

```diff
@@ -130,19 +130,19 @@ newtype ExistenceResult = ExistenceResult
   } deriving (Show, Eq)
 
 newtype PossiblyIncompleteResults = PossiblyIncompleteResults
-  { possiblyIncompleteResults :: HashMap RequestKey CommandResult
+  { possiblyIncompleteResults :: HashMap RequestKey (CommandResult [TxLog Value])
   } deriving (Show, Eq)
 
 data ListenerResult =
-  ListenerResult CommandResult |
+  ListenerResult (CommandResult [TxLog Value]) |
```

Comment:
Since the API takes `Hash` we should consider using that in `History`, nobody will ever need the logs

Reply:
https://github.com/kadena-io/pact/pull/504/files/7c67835ca97d6f9c5479bce0796ae524827d39f5#diff-286d1c1c88fd6b4b5ff21b71335bbc9eR40
Should we make PactService also output `Hash` then?

Reply:
I think so, it just gets messaged to the API

Reply:
Commit https://github.com/kadena-io/pact/pull/504/commits/b36c40328096a6915b32e9bc2e00b64434eca2f6
makes it so that `CommandResult Hash` is used everywhere in the pact server.